fix: soften 'never' in pointer-events:none description (fixes #44565) (#44566)
mdn/content: files/en-us/web/css/reference/properties/pointer-events/index.md

- `auto`
  - : The element behaves as it would if the `pointer-events` property were not specified. In SVG content, this value and the value `visiblePainted` have the same effect.
- `none`

>  - : The element on its own is generally not the [target](/en-US/docs/Web/API/Event/target) of pointer events. However its subtree could be kept targetable by setting `pointer-events` to some other value. In these circumstances, pointer events will trigger event listeners on this parent element as appropriate on their way to or from the descendant during the event capture and [bubble](/en-US/docs/Web/API/Event/bubbles) phases. Note that `pointerenter` and `pointerleave` events are exceptions — they will still fire on a parent with `pointer-events: none` when the pointer moves into or out of a descendant that is hit-testable.

Reviewer: @nihalakp Thanks for this.
1. If you plan on fixing your own issue you don't need to create an issue - just a PR.  However if you do create an issue then you should usually wait for triage.
2. I think you're correct - "never" is wrong. I suspect "on its own" was supposed to capture that an element with no descendents would not be a target, but it misses that there are genuine exceptions in `pointerenter` and `pointerleave`.
3. My problem with this is that there is way too much duplication in having your sentence _"Note that `pointerenter` ..._ immediately followed by the note, which covers much the same ground.
   A possible fix is to mention the exception up front and then rely on the note - see below. 

```suggestion
 - : The element is generally not the [target](/en-US/docs/Web/API/Event/target) of pointer events, with the exception of `pointerenter` and `pointerleave` events fired at the element or its descendents.
   Note that pointer events targeted at descendents (that don't set `pointer-events` to `none`) will still trigger event listeners on this parent element during the event capture and [bubble](/en-US/docs/Web/API/Event/bubbles) phases.
```

Happy for you to propose an alternative that removes the duplication if this isn't quite correct.
